Common subdirectories: pxmlutils_0.0.9.original/.arch-ids and pxmlutils_0.0.9/.arch-ids
diff -u pxmlutils_0.0.9.original/pxml3.cl pxmlutils_0.0.9/pxml3.cl
--- pxmlutils_0.0.9.original/pxml3.cl	2004-01-15 10:57:53.000000000 -0600
+++ pxmlutils_0.0.9/pxml3.cl	2004-01-27 21:49:23.620276504 -0600
@@ -146,7 +146,7 @@
 
 (defun next-dtd-token (tokenbuf
 		       external include-count external-callback)
-  (declare (:fbound parse-default-value) (optimize (speed 3) (safety 1)))
+  (declare #+allegro (:fbound parse-default-value) (optimize (speed 3) (safety 1)))
   (macrolet ((add-to-entity-buf (entity-symbol p-value)
 	       `(progn
 		  (push (make-tokenbuf :cur 0 :max (length ,p-value) :data ,p-value)
@@ -2395,7 +2395,7 @@
     ))
 
 (defun external-param-reference (tokenbuf old-coll external-callback)
-  (declare (:fbound next-token) (ignorable old-coll) (optimize (speed 3) (safety 1)))
+  (declare #+allegro (:fbound next-token) (ignorable old-coll) (optimize (speed 3) (safety 1)))
   (setf (iostruct-seen-parameter-reference tokenbuf) t)
   (macrolet ((add-to-entity-buf (entity-symbol p-value)
 	       `(progn
Only in pxmlutils_0.0.9: pxml3.cl~
Common subdirectories: pxmlutils_0.0.9.original/{arch} and pxmlutils_0.0.9/{arch}
